src/main/scala/com/mongodb/spark/api/java/sql/fields/Binary.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.spark.api.java.sql.fields;
+
+import org.bson.BsonBinarySubType;
+
+import java.util.Arrays;
+
+/**
+ * A Java bean representing the Bson Binary type
+ *
+ * @since 1.0
+ */
+public final class Binary {
```

Comment:
It's not clear to why org.bson.BsonBinary couldn't be used instead of this new class, as they are nearly identical.   Is it because org.bson.BsonType leaks out as a property value?